server.ts

```diff
@@ -33,6 +33,31 @@ app.post('/api/graphql', async (req, res) => {
     return;
 });
 
+type Error = {
+    statusCode: number,
+    message: string,
+    __code: number
+}
+
+app.post('/api/getSignaturesForAddress', async(req, res, next) => {
+    try{
```

Comment:
I think you might need to run the ESLint and Prettier scripts in the `package.json`. There's also a script to auto-format anything that's non-compliant. Mind formatting this PR?